src/plan.rs

```diff
@@ -90,24 +90,25 @@ impl InstallPlan {
         let buf = format!(
             "\
             Nix install plan (v{version})\n\
-            \n\
-            Planner: {planner}\n\
+            Planner: {planner} {maybe_default_setting_note}\n\
```

Comment:
```suggestion
            Planner: {planner}{maybe_default_setting_note}\n\
```

The note already includes the space, no need for two ;)